setup.cfg

```diff
@@ -43,7 +43,7 @@ install_requires =
     pyuvdata>=2.0<3.0
     astropy_healpix
     astropy
-    vis_cpu>=0.1.1
+    vis_cpu
```

Comment:
I would leave this with the >=, since we really do need that version or greater.

Reply:
Thanks, I've actually bumped this to >=0.2.0, since this PR uses recent changes in vis_cpu